### PR TITLE
fix latest tagging to tag only on git-tags refs

### PR DESCRIPTION
**Pull request by vrunoa**

### CODE REVIEW

File: .github/workflows/release.yml [hunk 1]
```diff
@@ -2,23 +2,38 @@ name: Docker Release
...
 on:
   push:
-    branches:
-      - master
     tags:
-      - v*
+      - 'v*.*.*'
 
-jobs:
+jobs:
```
Comment (christian-bromann) [inline .github/workflows/release.yml:8]:
```suggestion
jobs:
```

File: .github/workflows/release.yml [hunk 2]
```diff
@@ -2,23 +2,38 @@ name: Docker Release
...
   release:
     runs-on: ubuntu-latest
     steps:
       -
         name: Checkout
         uses: actions/checkout@v2
       -
-        uses: docker/build-push-action@v1
+        name: Set up Docker Buildx
+        uses: docker/setup-buildx-action@v1
```
Comment (christian-bromann) [inline .github/workflows/release.yml:17]: Is there anything different with the testcafe image that we need Docker Buildx?

Comment (vrunoa) [reply .github/workflows/release.yml:17]: @christian-bromann this is needed by github actions, otherwise it fails to publish.

**Review by christian-bromann — APPROVED** [review]
👍

**Pull request merged**